cnn_convertor/cnn_layer.py

```diff
@@ -21,8 +21,14 @@
 from enum import IntEnum, auto
 
 
-def get_conv_out_width(width, kx, pad_left, pad_right, stride):
-    return (pad_left + width + pad_right - kx) // stride + 1
+def get_conv_out_width(width, kx, pad_left, pad_right, stride, dilation):
+    return (pad_left + width + pad_right - ((kx - 1) * dilation + 1))\
+            / stride + 1
```

Comment:
I think it should be remained to use `// stride` since in Python / will return float result.

Reply:
OK, I see that you added another function to get the integer result instead. Ignore my first comment.